src/components/__tests__/__snapshots__/publicAPI.spec.js.snap

```diff
@@ -10930,10 +11444,122 @@ Map {
       },
     },
   },
+  "ToolbarDivider" => Object {
```

Comment:
Have you looked into these `Toolbar`-related components and checked if they will conflict with our `Toolbar`?

Reply:
Good question! It seems like we'll export our `Toolbar` and not Carbon's, which as it stands, appears to be deprecated anyway - https://react.carbondesignsystem.com/?path=/story/toolbar-deprecated--default

Reply:
Sounds good -- thank you!